src/settings.cpp

```diff
@@ -630,6 +647,17 @@ void Settings::LoadDefaultsOrSave(std::string path)
  	settings[XORSTR("AutoKnife")][XORSTR("Filters")][XORSTR("allies")] = Settings::AutoKnife::Filters::allies;
  	settings[XORSTR("AutoKnife")][XORSTR("onKey")] = Settings::AutoKnife::onKey;
 
+	settings[XORSTR("QuickSwitch")][XORSTR("enabled")] = Settings::QuickSwitch::enabled;
+
+	settings[XORSTR("LeftKnife")][XORSTR("enabled")] = Settings::LeftKnife::enabled;
+
+	// Debug cfgs
+	settings[XORSTR("Debug")][XORSTR("AutoWall")][XORSTR("debugView")] = Settings::Debug::AutoWall::debugView;
```

Comment:
we don't save the debug options because they are prone to lag/crash

Reply:
Well yes. I'm going to revert this